### PR TITLE
Ignore downloading exercise videos when `projector_key` is None

### PR DESCRIPTION
**Pull request by atlonxp**
The downloader is error when the `projector_key` is None, causing failure in the following lines:

```
video = self._get_video(exercise.data.get("projector_key")) # --> video = None
...
if videos and video.video_mp4_link # --> NoneType has no video_mp4_link
```

### CODE REVIEW

**Comment by TRoboto** [comment]
Have you encountered any issue with it? If so please provide with which course.

File: src/datacamp_downloader/datacamp_utils.py [hunk 1]
```diff
@@ -288,44 +288,46 @@ def download_others(self, course_id, chapter: Chapter, path: Path, **kwargs):
...
+                        )
+                    if audios and video.audio_link:
+                        download_file(
+                            self.session,
+                            video.audio_link,
+                            path / "audios" / f"ch{chapter.number}_{video_counter}.mp3",
+                            False,
+                            overwrite=self.overwrite,
+                        )
+                    if scripts and video.script_link:
```
Comment (TRoboto) [inline src/datacamp_downloader/datacamp_utils.py:310]: For cleaner code, you should revert your changes and add only the following:
`
if not video:
    continue
`

**Comment by atlonxp** [comment]
2003 if I can remember. It is a random course ID, and it works fine until I found the error.

Have a look. i have also asked for pull request. 

There is another one when exercise cannot be pulled (aka None)

**Review by TRoboto — CHANGES_REQUESTED** [review]

**Comment by atlonxp** [comment]
changed as your recommended

**Review by TRoboto — APPROVED** [review]

**Comment by TRoboto** [comment]
Thank you for your contribution.

**Pull request merged**